doc/man3/DTLS_set_timer_cb.pod

```diff
@@ -0,0 +1,36 @@
+=pod
+
+=head1 NAME
+
+DTLS_set_timer_cb - Set callback for controlling DTLS timer duration
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ void DTLS_set_timer_cb(SSL *s, unsigned (*cb)(SSL *s, unsigned timer));
+
+=head1 DESCRIPTION
+
+This function sets an optional callback function for controlling the
+timeout interval on the DTLS protocol. The callback function will be
+called by DTLS for every new DTLS packet that is sent.
```

Comment:
We need to describe how the callback function actually works here, and that what it means when a timer value of 0 is passed.